### ISSUE TITLE
branch: accessible-instance

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/joist#305]
While adding accessible content to HomeScreenView, an issue was found involving the removal of accessible instances.  This branch holds some accessibility related changes to HomeScreenView so that we can easily reproduce the issue.  As soon as the issue is resolved, this branch should be deleted.

The issue was tracked to Node.removeChildWithIndex so documentation on the issue will exist in the scenery repository.

**Comment by jessegreenberg** [comment]
See https://github.com/phetsims/scenery/issues/491 for documentation on the issue.

**Comment by jonathanolson** [comment]
Resolved issue, assigning to @jessegreenberg to handle merge.

**Comment by jessegreenberg** [comment]
I completed the merge, including the accessible content, and a lint fix that @jonathanolson added.  phetsims/scenery#491 is resolved.  This branch is no longer necessary, closing.
